scene/3d/physics/vehicle_body_3d.cpp

```diff
@@ -267,6 +267,11 @@ Node3D *VehicleWheel3D::get_contact_body() const {
 	return m_raycastInfo.m_groundObject;
 }
 
+//Return the normal force magnitude at contact point along contact_normal.
```

Comment:
```suggestion
// Return the normal force magnitude at contact point along contact_normal.
```
Or
```suggestion
```
It's not very informative